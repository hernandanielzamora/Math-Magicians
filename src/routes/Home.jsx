import React from 'react';
import Welcome from '../components/Welcome';

const Home = () => (
  <>
    <Welcome />
  </>
);

export default Home;
