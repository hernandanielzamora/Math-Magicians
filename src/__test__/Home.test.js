import { render } from '@testing-library/react';
import Home from '../routes/Home';

describe('Tests for the Home page', () => {
  it('Should render the Home page properly', () => {
    const quotesPage = render(<Home />);
    expect(quotesPage).toMatchSnapshot();
  });
});
