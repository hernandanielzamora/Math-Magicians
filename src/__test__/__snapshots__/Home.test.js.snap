// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for the Home page Should render the Home page properly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="welcomeContainer"
      >
        <h2
          class="welcomeTitle"
        >
          Welcome to our page!
        </h2>
        <p
          class="welcomeText"
        >
          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nulla, consequuntur incidunt beatae vitae quibusdam, ut iusto, aliquid possimus id pariatur tempora minima debitis vero? Corporis, error in! Eius, autem sapiente. Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nulla, consequuntur incidunt beatae vitae quibusdam, ut iusto, aliquid possimus id pariatur tempora minima debitis vero? Corporis, error in! Eius, autem sapiente. Lorem ipsum dolor sit amet consectetur adipisicing elit. Vel magni maiores tempora, quisquam autem fugit laboriosam odit quam repellat suscipit hic provident explicabo mollitia sunt ducimus possimus error! Non, quia! Lorem ipsum dolor sit amet consectetur adipisicing elit.
        </p>
        <p
          class="welcomeText"
        >
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Vel magni maiores tempora, quisquam autem fugit laboriosam odit quam repellat suscipit hic provident explicabo mollitia sunt ducimus possimus error! Non, quia! Lorem ipsum dolor sit amet consectetur adipisicing elit. Vel magni maiores tempora, quisquam autem fugit laboriosam odit quam repellat suscipit hic provident explicabo mollitia sunt ducimus possimus error! Non, quia! Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nulla, consequuntur incidunt beatae vitae quibusdam, ut iusto, aliquid possimus id pariatur tempora minima debitis vero? Corporis, error in! Eius, autem sapiente.
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="welcomeContainer"
    >
      <h2
        class="welcomeTitle"
      >
        Welcome to our page!
      </h2>
      <p
        class="welcomeText"
      >
        Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nulla, consequuntur incidunt beatae vitae quibusdam, ut iusto, aliquid possimus id pariatur tempora minima debitis vero? Corporis, error in! Eius, autem sapiente. Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nulla, consequuntur incidunt beatae vitae quibusdam, ut iusto, aliquid possimus id pariatur tempora minima debitis vero? Corporis, error in! Eius, autem sapiente. Lorem ipsum dolor sit amet consectetur adipisicing elit. Vel magni maiores tempora, quisquam autem fugit laboriosam odit quam repellat suscipit hic provident explicabo mollitia sunt ducimus possimus error! Non, quia! Lorem ipsum dolor sit amet consectetur adipisicing elit.
      </p>
      <p
        class="welcomeText"
      >
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Vel magni maiores tempora, quisquam autem fugit laboriosam odit quam repellat suscipit hic provident explicabo mollitia sunt ducimus possimus error! Non, quia! Lorem ipsum dolor sit amet consectetur adipisicing elit. Vel magni maiores tempora, quisquam autem fugit laboriosam odit quam repellat suscipit hic provident explicabo mollitia sunt ducimus possimus error! Non, quia! Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nulla, consequuntur incidunt beatae vitae quibusdam, ut iusto, aliquid possimus id pariatur tempora minima debitis vero? Corporis, error in! Eius, autem sapiente.
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
